tagged_data.go

```diff
@@ -0,0 +1,134 @@
+package iotago
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/iotaledger/hive.go/serializer/v2"
+	"github.com/iotaledger/iota.go/v3/util"
+)
+
+const (
+	// TaggedPayloadTagMaxLength defines the max length of the tag within a TaggedData payload.
+	TaggedPayloadTagMaxLength = 64
+	// TaggedPayloadTagMinLength defines the min length of the tag within a TaggedData payload.
+	TaggedPayloadTagMinLength = 0
```

Comment:
Does the `TaggedPayloadTagMinLength` check make any sense if it zero?

Reply:
1c56bd813d8ec4a10c7e676d06cbed2f3c8ade0a